test/helpers/request.js

```diff
@@ -0,0 +1,20 @@
+const request = require('supertest');
+const nock = require('nock');
+const ServiceUrlGetter = require('../../services/service-url-getter');
+
+const urlService = new ServiceUrlGetter().perform();
+
+nock(urlService)
+  .persist()
+  .get('/liana/ip-whitelist-rules')
+  .reply(200, {
+    data: {
+      type: 'ip-whitelist-rules',
+      id: '1',
+      attributes: {
+        rules: [],
```

Comment:
it would be closer to the protocol to set `use_ip_whitelist: false`.
It would prevent useIpWhitelist to be `undefined` during the tests, which works as the false value.